va/va.h

```diff
@@ -3922,6 +3922,38 @@ VAStatus vaQuerySurfaceError(
  */
 #define VA_FOURCC_IYUV          0x56555949
 
+/** Y8: 8-bit greyscale.
+ */
+#define VA_FOURCC_Y8            0x20203859
```

Comment:
Is it possible for us to add the FourCC explanation as the above definition? Then, it is clear for the readers to know which FourCC. Thanks a lot in advance!